kaldb/src/test/java/com/slack/kaldb/chunk/IndexingChunkImplTest.java

```diff
@@ -531,7 +531,7 @@ public void testSnapshotToS3UsingChunkApi() throws Exception {
       assertThat(chunk.snapshotToS3(bucket, "", s3BlobFs)).isTrue();
       assertThat(chunk.info().getSnapshotPath()).isNotEmpty();
 
-      assertThat(getCount(INDEX_FILES_UPLOAD, registry)).isEqualTo(4);
+      assertThat(getCount(INDEX_FILES_UPLOAD, registry)).isEqualTo(15);
```

Comment:
I think this change should also be undone since we use compound files now.

Reply:
Done. This was a rebase issue.